01_02_Combat_Rules.txt

```diff
@@ -999,3 +999,128 @@ retrieve a dropped weapon or stand back up if they have fallen.
+every room in a structure is called "clearing a building." 
+
+    What constitutes a "room" that can be breached and cleared is ultimately
+up to your DM, but usually a room that can be cleared is smaller than 20m by
+20 meters and has a discrete number of entry points. For example, a spaceship
+hangar that is 35m by 18m and has a service door on either side when the hangar
+door is shut would allow for stacking up with the DM's permission. That same 
+hangar with the hangar door open would need to be maneuvered into and fought 
+over normally since the hangar door is most of one of its walls. Another example
+would be that an open-air pagoda probably wouldn't be breach-able because it 
+lacks any door to stack up on and has no walls.
+
+    What constitutes an "entry point" is also up to the DM. These are usually
+unlocked doorways, but can also be skylights to rappel from, windows large 
+enough to step through once smashed, or chunks of wall that have been blown up.
+An area cannot be an entry point if it cannot be moved through by the character
+stacked up on it at the time of the breach. A character can only stack up on an
+entry point to a room.
+
+    In Compound X, the process of clearing a room follows the three steps listed 
+below.
+
+1. Stacking up
+2. Breaching
+3. Clearing
+
+===== Stacking Up ===== 
+
+    Characters can join a stack up line on any time during their turn. Doing so
+immediately ends their turn and the rest of their actions are considered held
+for use during clearing. When a character stacks up, they must indicate exactly
+one entry point that they will use during the breach. If there is any ambiguity,
+they must indicate which room they will storm when the breach occurs. Which room
+you are breaching is usually obvious; it's the room on the other side of the 
+entry point from where your character is stacking up. On the rare occasion that
+one door leads to more than one room, a single room must be designated for 
+clearing. You cannot breach and clear a room that is not attached to the entry
+point that you stacked up on. 
+
+    If your turn starts and you are already stacked up, you may either leave the
+line of the stack up and take your turn normally, or skip your turn to remain in
+the stack up line. If you skip your turn in this manner, you will have the same
+number of actions during clearing as you would if you stacked up as the first
+act of your turn.
+
+    Any number of characters can stack up on a given entry point. The order of
+the characters in the lineup is important. Players can chose any position in the
+stackup order that hasn't yet been taken at the time that they declare their 
+character will stack up. So the first character to stack up could choose to get
+in the middle of the line, and the second character to use their turn to stack
+up could stack up in front if they so chose.  
+
+===== Breaching =====
+
+    Any time a character joins the stack up line, the character in the lead of
```

Comment:
Brilliant use of a very specific and highly relevant trigger condition. The 🐢 approves.

Reply:
thanks! :)